en/lessons/introduction-to-populating-a-website-with-api-data.md

```diff
@@ -0,0 +1,529 @@
+---
+title: |
+    Introduction to Populating a Website with API Data
+collection: lessons
+layout: lesson
+slug: website-api
+date: 2019-05-22
+authors:
+- Go Sugimoto
+reviewers:
+reviewers:
+- Amber Stubbs
+- Brianna Allen
+- Adam Crymble
+editors:
+- Jessica Parr
+review-ticket: https://github.com/programminghistorian/ph-submissions/issues/196
+difficulty: 2
+activity: acquiring
+topics: apis
```

Comment:
`topics` is like `authors` - it needs to be supplied as a list variable. In this case, please change it to

```yaml
topics:
- api
```